app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -76,8 +91,8 @@ def send_success_response
                        message: "Application created" }, status: :ok
       end
 
-      def send_failed_response
-        render json: { message: "Unable to create application" },
+      def send_failed_response(message = nil)
+        render json: { message: message || "Unable to create application" },
```

Comment:
it sounds like ||= is a safer bet per all the documentation - as it will never throw a "NameError" like || can - but as this is fixing something that's already broken anyway any change is an improvement so I won't push for it.

Reply:
(probably we can get @benbaumann95  to also review as he's done far more error catching then I have ;) ) -

Reply:
It's not a safer bet, as per your link:

>  The difference is that, when a is undefined, `a || a = b` would raise `NameError`, whereas `a ||= b` sets `a` to `b`.

We' not using either of these syntaxes because we're not assigning anything at all on this line. `messages` will always be defined, and there's no point reassigning it in the middle of another expression - if anything it creates confusion as it's not always true that assignments return their value (Ruby does for that matter, but it doesn't make it an obvious assumption).